### PR TITLE
Homework1/sandra mojsova

### PR DESCRIPTION
**Pull request by SandraMojsova**

### CODE REVIEW

File: homeworks/homework-02/basic-table.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <title>Basic Table</title>
+</head>
+<body>
+    <main>
+        <table border="1">
+            <thead>
+                <tr>
+                   <th colspan="3">section_1</th>
+                </tr>
+            </thead>
```
Comment (dpeshevski) [inline homeworks/homework-02/basic-table.html:16]: ne ti treba `<thead>` spored slikata imash samo `<tbody>`

**Review by meto2208 — APPROVED** [review]
Very good.

**Review by Metodija-Mitovski — APPROVED** [review]
good work

File: homeworks/homework-02/basic-table.html (hunk 1 above)
Comment (SandraMojsova) [reply homeworks/homework-02/basic-table.html:16]: Da , greska sum procenila . Greskata e ispravena🙂 , <thead> elementot e ostranet.

**Review by emillija — APPROVED** [review]
super odlicno

**Pull request merged**